app/main/routes.py

```diff
@@ -84,14 +84,64 @@ def callback():
     session["token_scope"] = access_token_response["scope"]
     session["session_state"] = access_token_response["session_state"]
 
-    return redirect(url_for("main.poc_search"))
+    return redirect(url_for("main.browse"))
 
 
 @bp.route("/accessibility", methods=["GET"])
 def accessibility():
     return render_template("accessibility.html")
 
 
+@bp.route("/browse", methods=["POST", "GET"])
+@access_token_sign_in_required
+def browse():
+    transferring_body_id = request.args.get("transferring_body", None)
+    series_id = request.args.get("series", None)
+    consignment_id = request.args.get("consignment_id", None)
```

Comment:
can we make this consistent with the other 2 args either drop _id on this one or add _id to those

Reply:
Added _id to keep consistent with the rest of the code.